neurobooth_os/tasks/task.py

```diff
@@ -25,7 +26,22 @@
 from neurobooth_os.log_manager import APP_LOG_NAME
 import neurobooth_os.config as cfg
 
-from enum import Enum
+from pylsl import local_clock
+
+
+class TaskAborted(Exception):
+    """Exception raised when the task is aborted."""
+    pass
+
+
+def check_if_aborted(keys=("q",)) -> None:
+    """
+    Check to see if a task has been aborted. If so, raise an exception.
+    :param keys: The keys that will abort a task.
+    """
+    if event.getKeys(keyList=keys):
```

Comment:
What does the variable rename (keys to keyList) do? Does getKeys() modify the argument? 

I'm wondering if this couldn't be simplified. Why does the method take an argument, if it's not called with an argument. It seems like the 'quit key' should be a constant 'q' (or '(q,)' if getKeys() requires a list).

Reply:
It's not a rename, it's specifiying a keyword argument by name. getKeys() does require a list. (See https://psychopy.org/api/event.html):

> keyList: None or []
> Allows the user to specify a set of keys to check for. Only keypresses from this set of keys will be removed from the keyboard buffer. If the keyList is None, all keys will be checked and the key buffer will be cleared completely. NB, pygame doesn’t return timestamps (they are always 0)